subsys/nrf_security/src/psa_crypto_driver_wrappers.c

```diff
@@ -809,6 +846,15 @@ psa_status_t psa_driver_wrapper_derive_key(const psa_key_attributes_t *attribute
 	switch (PSA_KEY_LIFETIME_GET_LOCATION(attributes->lifetime)) {
 	case PSA_KEY_LOCATION_LOCAL_STORAGE:
 		/* Add cases for transparent drivers here */
+#ifdef PSA_CRYPTO_DRIVER_IRONSIDE
+		status = ironside_psa_derive_key(attributes, input, input_length, key_buffer,
```

Comment:
First is that it would be nice to use PSA_NEED_ prefixed symbols here. You only need one symbol for key management and a couple more for derivation/pake.

Secondly, is this plan to be used at the same time with other drivers or used by default?